### DISCUSSION TITLE
Libraries updated.

### DISCUSSION THREAD

**Discussion by ericwooley** [ericwooley/WeMips#13] [Announcements]
@ralfgerlich 

I updated the libraries, to get rid of some really bad XSS stuff that was probably possible.

Bootstrap has been updated to [3.4.1](https://security.snyk.io/package/npm/bootstrap/3.4.1), underscore to [1.13.6](https://security.snyk.io/package/npm/underscore/1.13.6), jquery to [1.12.4](https://security.snyk.io/package/npm/jquery/1.12.4).

[Code mirror](https://security.snyk.io/package/npm/codemirror/3.19.0) doesn't have anything real bad, so I just left it alone, since I think that might be a huge pain to update.

I tested out various functionality, and everything seems to be working. But LMK if you have issues due to the upgrades.

**Comment by ralfgerlich** [comment]
@ericwooley Thank you for that. So far everything seems fine.
